config/app.php

```diff
@@ -234,4 +234,8 @@
     'task_drafts_enabled' => env('TASK_DRAFTS_ENABLED', true),
 
     'force_https' => env('FORCE_HTTPS', true),
+
+    // Process Request security log rate limit: 1 per day (86400 seconds)
+    'process_request_errors_rate_limit' => env('PROCESS_REQUEST_ERRORS_RATE_LIMIT', 1),
```

Comment:
Add `PROCESS_REQUEST_ERRORS_RATE_LIMIT` to .env.example file

Reply:
Fixed